app/controllers/posts_controller.rb

```diff
@@ -539,7 +539,11 @@ def notice
   def destroy_bookmark
     params.require(:post_id)
 
-    bookmark_id = Bookmark.where(post_id: params[:post_id], user_id: current_user.id).pluck_first(:id)
+    bookmark_id = Bookmark.find_by(
```

Comment:
Is there a reason why`&.id` is preferred over `.pluck_first(:id)`?

Reply:
Hmm I don't think so, will change to pluck_first :)